sale_order_line_input/README.rst

```diff
@@ -0,0 +1,21 @@
+**This file is going to be generated by oca-gen-addon-readme.**
+
+*Manual changes will be overwritten.*
```

Comment:
Make this file minimal for less diff when generated. Let only the first sentence.